### PR TITLE
Remove unused variable

### PR DESCRIPTION
**Pull request by danghai**
I make PR to remove some unused variable in script.

### CODE REVIEW

File: scripts/whisper_log_trace_csv.py [hunk 1]
```diff
@@ -33,7 +33,7 @@
 
 LOGGER = logging.getLogger()
 
-def process_whisper_sim_log(whisper_log, csv, full_trace = 0):
```
Comment (taoliug) [inline scripts/whisper_log_trace_csv.py]: Please keep the full_trace as the functionality will be added later.

File: scripts/whisper_log_trace_csv.py [hunk 2]
```diff
@@ -86,7 +85,7 @@ def main():
   args = parser.parse_args()
   setup_logging(args.verbose)
   # Process whisper log
-  process_whisper_sim_log(args.log, args.csv, args.full_trace)
```
Comment (taoliug) [inline scripts/whisper_log_trace_csv.py]: same here

File: cov.py [hunk 3]
```diff
@@ -33,7 +33,7 @@
 
 def collect_cov(log_dir, out, core, iss, testlist, batch_size, lsf_cmd, steps, \
                 opts, timeout, simulator, simulator_yaml, custom_target, \
-                isa, target, stop_on_first_error,
```
Comment (taoliug) [inline cov.py]: Probably keep this, we may use it later

File: run.py [hunk 4]
```diff
@@ -429,14 +428,13 @@ def iss_sim(test_list, output_dir, iss_list, iss_yaml, isa, setting_dir, timeout
           logging.debug(cmd)
 
 
-def iss_cmp(test_list, iss, output_dir, isa, stop_on_first_error):
```
Comment (danghai) [inline run.py]: What about this one?

Comment (taoliug) [reply run.py]: This can be removed.

**Comment by taoliug** [comment]
Please resolve conflicts.

**Comment by danghai** [comment]
@taoliug Sorry for late response. I resolve the conflict. Thank you

**Pull request merged**